src/event/event_kevent.c

```diff
@@ -528,11 +534,17 @@ _dispatch_kevent_merge_muxed(dispatch_kevent_t ke)
 	}
 }
 
+#define DISPATCH_KEVENT_ERSATZ_EVFILT_USER_MASK 0xfffffe00
```

Comment:
Does it make sense to have a top level macro for the use of this?  Something similar to `DISPATCH_USE_KEVENT_ERSATZ_USER_FILTER`?

Reply:
I renamed it to `DISPATCH_KEVENT_ERSATZ_EVFILT_USER_IDENT`, since this isn't really being used as a mask. The "use" verb doesn't really apply here, because we check `#ifdef EVFILT_USER` -- if there's no EVFILT_USER, the platform doesn't support this filter type and should use the ersatz implementation.